mscore/scoreview.h

```diff
@@ -334,6 +333,7 @@ class ScoreView : public QWidget, public MuseScoreView {
       virtual void cmdAddSlur(ChordRest*, ChordRest*) override;
       virtual void cmdAddHairpin(HairpinType);
       void cmdAddNoteLine();
+      void cmdGotoElement(Element*);
```

Comment:
We could probably move handling `"get-location"` command handling to `ScoreView::cmd`, then there will be no need in making `cmdGotoElement` public.

Reply:
Good point.  The only reason it is in musescore.cpp now is originally I tested the code by associating it FWIW, with the zoom100 command, and I hadn't figured out yet I would need cmdGotoElement().

FWIW, I *could* move it back to zoom100 and avoid the translation string, but that's kind of hacky, no problem waiting until 3.4

Reply:
Could you please move this command to ScoreView then? It looks like this would be a cleaner way to implement this command.